packages/editor/src/lib/exports/html-to-image/clonePseudos.ts

```diff
@@ -0,0 +1,56 @@
+import { uniqueId } from '@tldraw/utils'
+import { embedCssValueUrlsIfNeeded, shouldIncludeCssProperty } from './embedCss'
+
+type Pseudo = ':before' | ':after'
+
+async function formatCSSProperties(style: CSSStyleDeclaration) {
+	let cssText = ''
+	for (const property of style) {
+		if (!shouldIncludeCssProperty(property)) continue
+		let value = style.getPropertyValue(property)
+		const priority = style.getPropertyPriority(property)
+
+		const replaced = embedCssValueUrlsIfNeeded(value)
+		if (replaced) {
+			value = await replaced
+		}
+
+		cssText += `${property}: ${value}${priority ? ' !important' : ''};`
+	}
+	return cssText
+}
+
+async function getPseudoElementStyle(
+	className: string,
+	pseudo: Pseudo,
+	style: CSSStyleDeclaration
+): Promise<Text> {
+	const selector = `.${className}:${pseudo}`
+	const cssText = await formatCSSProperties(style)
+
+	return document.createTextNode(`${selector}{${cssText}}`)
+}
+
+async function clonePseudoElement(node: HTMLElement, pseudo: Pseudo) {
+	const style = window.getComputedStyle(node, pseudo)
```

Comment:
e.g. here it's just `:before` instead of `::before` - i'm sure it works just fine, but technically it should be `::`, eh?